### DISCUSSION TITLE
[PG 11.2] Grafana 6.0.0-beta 3 - Numeric columns not displaying in dropdown

### DISCUSSION THREAD

**Discussion by FunDeckHermit** [grafana/grafana#37009] [Data Sources]
**What happened**:
I just installed grafana to connect to my postgresql database. 
Installed the 6.0.0 beta3 version because I hoped postgreSQL 11 would be supported.

Created my first dashboard, added a graph and tried to select a Numeric(5,3) column. Only integer columns were displayed in the dropdown. 

Manual adding of the column name displays the graph without any problems. Problem is just the dropdown. 

**What you expected to happen**:
A dropdown containing all columns (numeric and integers).

**How to reproduce it (as minimally and precisely as possible)**:
Install grafana 6.0.0 beta3
install postgresql 11.2
create table with numeric values.
Add dashboard, graph and select column. 

**Anything else we need to know?**:
Table is partitioned using native postgresql 11 partitioning. 

**Environment**:
- Grafana version: 6.0.0 beta3
- Data source type & version: PostgreSQL 11.2
- OS Grafana is installed on: Ubuntu 18.04
- User OS & Browser: Firefox
- Grafana plugins: none
- Others: none

**Comment by torkelo** [comment]
@svenklemm any ideas?

**Comment by marefr** [comment]
@mindfuucker we currently don't have any plans working on this. If you're interested in digging into what may be the problem and/or contribute a fix I think the related [line of code](https://github.com/grafana/grafana/blob/master/public/app/plugins/datasource/postgres/meta_query.ts#L130) in this, but I'm not 100% sure.

Thanks

**Comment by nick-bowman** [comment]
This issue was encountered again and described in #43444 and there is a proposed fix in #43445 waiting for approval. Would someone be able to review the proposed PR?

**Reply by zoltanbedi** [reply]
Commented on your pr.

**Comment by gabor** [comment]
based on the info here, this has been fixed in https://github.com/grafana/grafana/pull/43445, closing the discussion.
(if this did not fix it, please create a new github issue)
